### PR TITLE
React native app blog post

### PR DESCRIPTION
**Pull request by lmlynik**

### CODE REVIEW

File: 20180619/README.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
+# Quick and easy way to start with Matchmore: React-native
+
+We already showed you how to do it in Swift iOS, now it is time for the new kid on the block: _React-Native_.
```
Comment (adamsmo) [inline 20180619/README.md]: `Swift iOS` => `Swift for iOS`

File: 20180619/README.md [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+After this you will have a good understanding how to make your own Dating App with Matchmore!
+
+Before starting, it will be beneficial to have some React Native know-how before. And we will use Yarn!
+
+Lets start of with creating a React-Native app
```
Comment (adamsmo) [inline 20180619/README.md]: Let's start off with

File: 20180619/README.md [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+
+```
+react-native init DatingApp
```
Comment (adamsmo) [inline 20180619/README.md]: how do I get `react-native` ?

File: 20180619/README.md [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+## Get API key
+
+Now let's create a new application on Matchmore Portal and obtain the API key that we will use to configure SDK later on.
+After logging in ([register for free here](http://matchmore.com/account/register/)) you do the following:
+
+![alt text](https://raw.githubusercontent.com/matchmore/tech-blog/master/20180329/img/create-app.gif "create app")
+
+## Setup persistence
+
+You need to wire in the persistence, of your choice.
```
Comment (adamsmo) [inline 20180619/README.md]: I do not know what this sentence means

File: 20180619/README.md (hunk 1 above)
Comment (maciejburda) [reply 20180619/README.md]: Maybe `in iOS using Swift.`

File: 20180619/README.md [hunk 5]
```diff
@@ -0,0 +1,359 @@
+# Quick and easy way to start with Matchmore: React-native
+
+We already showed you how to do it in Swift iOS, now it is time for the new kid on the block: _React-Native_.
+After this you will have a good understanding how to make your own Dating App with Matchmore!
+
+Before starting, it will be beneficial to have some React Native know-how before. And we will use Yarn!
```
Comment (maciejburda) [inline 20180619/README.md]: x2 before. I guess you can drop the second one.

File: 20180619/README.md [hunk 6]
```diff
@@ -0,0 +1,359 @@
...
+Now let's create a new application on Matchmore Portal and obtain the API key that we will use to configure SDK later on.
+After logging in ([register for free here](http://matchmore.com/account/register/)) you do the following:
+
+![alt text](https://raw.githubusercontent.com/matchmore/tech-blog/master/20180329/img/create-app.gif "create app")
+
+## Setup persistence
+
+You need to wire in the persistence, of your choice.
+
+It is small, for example for React-Native it would look like this
```
Comment (adamsmo) [inline 20180619/README.md]: what is small?

File: 20180619/README.md [hunk 7]
```diff
@@ -0,0 +1,359 @@
...
+```javascript
+let datingSubscription = await this.manager.createSubscription(
+      "Dating App", //topic specific to your applications, you can have multiple topic per app, it is up to you how will you model it, we will provide some insights about this soon
+      10000, //range of subscriptions in meters, lets start of with 10km
+      86400, //duration of how long the subscription will last, in seconds, lets put in 24 hours
+      "age >= 18" // matching selector
+    );
+```
+
+The matching selector is a selector which will filter out only publications which you are interested in. Think of it as SQL for publications you will receive! Here we are saying we are looking only for publications with a property `age` greater or equal.
```
Comment (adamsmo) [inline 20180619/README.md]: SQL table? SQL properties?

File: 20180619/README.md [hunk 8]
```diff
@@ -0,0 +1,359 @@
...
+      86400, //s
+      {age: 29, name: "Lucas"} //publication properties
+);
+```
+
+As we mentioned before, publication properties are the data we want to be searched by, our interests, picture url, age... whatever makes sense for your app.
+
+Ok, lets be real - we are just starting, nobody is our there, we need to create some "people" to test our and develop against.
+
+We can create virtual pins, which here we will mimic other people looking for their second half.
```
Comment (adamsmo) [inline 20180619/README.md]: `which here we will mimic other people` => `we will mimic with them other people` ?

File: 20180619/README.md [hunk 9]
```diff
@@ -0,0 +1,359 @@
...
+```
+
+When we run the app, we see
+
+![matches](https://raw.githubusercontent.com/matchmore/tech-blog/master/20180619/img/2-matches.png "matches")
+
+Ok, so what does it mean:
+
+- We get our own publication
+- We get we got some of them multiple times
```
Comment (adamsmo) [inline 20180619/README.md]: `We get we got some` ?

File: 20180619/README.md (hunk 7 above)
Comment (lmlynik) [reply 20180619/README.md]: I don't follow

File: 20180619/README.md (hunk 3 above)
Comment (lmlynik) [reply 20180619/README.md]: Not part of this blog post, your should know react native before reading this.

Comment (adamsmo) [reply 20180619/README.md]: 👌

**Pull request merged**